api/src/pcapi/core/categories/subcategories_v2.py

```diff
@@ -157,7 +157,7 @@ class NativeCategory(Enum):
     CARTES_JEUNES = "Cartes jeunes"
     EVENEMENTS_CINEMA = "Evènements cinéma"
     ARTS_VISUELS = "Arts visuels"
-    FESTIVAL_DU_LIVRE = "Festivals du livre"
+    FESTIVAL_DU_LIVRE = "Evénements autour du livre"
```

Comment:
On écrit évènement avec "è" partout (sauf aux endroits où l'on oublie régulièrement...) : 

```suggestion
    FESTIVAL_DU_LIVRE = "Évènements autour du livre"
```

... et je rajoute une majuscule accentuée parce que je suis comme ça. D'ailleurs, il manque aussi l'accent sur le "É" majuscule dans plusieurs labels dans ce fichier.